data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1075 @@
+			names "hai"
+			variant
+				"Shield Beetle (Particle Jump)"
+		fleet 5
+			names "hai"
+			variant
+				"Shield Beetle (Plasma Blaster Jump)"
+	npc
+		government "Human Alliance"
+		personality escort heroic waiting opportunistic
+		fleet "Human Wormhole Guards" 20
+	on complete
+		payment 600000
+
+
+mission "Hai Rescue [C05] Operatives"
+	landing
+	name "Pick up Operatives"
+	description "Head to <destination> to pick up 200 operatives and their 70 tons of gear. Payment is <payment>."
+	source "Foundry"
+	destination "Valhalla"
+	to offer
+		has "Hai Rescue [C04] Foundry: done"
+	on offer
+		conversation
+			`Deputy Director Stephen Moray greets you as you exit your ship. "The Unfettered are fearsome pilots, as are you, but we need some special operatives to storm the city. The Republic has rounded up the best troops in the whole of the Republic and has armed them with some special equipment. They're being very hush-hush about it, but I understand it is advanced anti-pirate gear Deep Sky has been developing."`
+			`	"Head to <destination> to pick up the 200 operatives and their 70 tons of gear. The Republic will give you your usual payment of <payment>."`
+			`	"I'll keep the Unfettered distracted... and secret... until you return."`
+				accept
+	on complete
+		payment 600000
+
+mission "Hai Rescue [C06] Meet the Fleet"
+	landing
+	name "Rejoin the Fleet at <planet>"
+	description "Rejoin the Unfettered at <destination>. Payment is <payment>."
+	source "Valhalla"
+	destination "Foundry"
+	passengers 200
+	cargo "anti-pirate gear" 70
+	blocked "You haven't the room for all <bunks> operatives and their <tons> of equipment. Land here again once you have that much space."
+	to offer
+		has "Hai Rescue [C05] Operatives: done"
+	on visit
+		dialog `You forgot some of the operatives or their equipment; they're probably in your escorts. Wait for the escorts to arrive and land again to finish this mission.`
+	on offer
+		log `Agent Kopkow of Republic Intelligence complained that they're running out of cover stories to hide the Hai. He needs the Hai-human conflict resolved without an more public displays. Once I have space for his 200 operatives, I need to pick them up at the Foundry spaceport.`
+		conversation
+			`Soon after landing, you are met by Agent Kopkow, the Republic Intelligence agent you met on Earth. He seems extremely concerned about something.`
+			`	"It is becoming increasingly difficult to cover up the existence of the Hai. The captains at Ultima Thule already know about the Hai, but we had to tell the crews we were dealing with a major pirate threat. We hired a few pirate gangs, tricking them into flying through the area to back our story. The Shield Beetles at Prime were explained away as experimental ship designs."`
+			`	"Our cover stories have passed the point of plausibility into the realm of barefaced lying. I hope you can finish this without too many more public displays, otherwise we won't be able to hide the Hai anymore.`
```

Comment:
```suggestion
			`	"It is becoming increasingly difficult to cover up the existence of the Hai. The captains at Ultima Thule already know about the Hai, but we had to tell the crews we were dealing with a major pirate threat. We hired a few pirate gangs, tricking them into flying through the area to back our story. The Shield Beetles at Prime were explained away as experimental ship designs. Our cover stories have passed the point of plausibility into the realm of barefaced lying. I hope you can finish this without too many more public displays, otherwise we won't be able to hide the Hai anymore.`
```